pkg/metrics/metrics.jsx

```diff
@@ -403,11 +409,11 @@ class CurrentMetrics extends React.Component {
 
         // top 5 CPU and memory consuming systemd units
         newState.topServicesCPU = n_biggest(this.cgroupCPUNames, this.samples[9], 5).map(
-            x => serviceRow(x[0], Number(x[1] / 10 / numCpu).toFixed(1)) // usec/s → percent
+            x => serviceRow(x[0], Number(x[1] / 10 / numCpu).toFixed(1), x[2]) // usec/s → percent
```

Comment:
This is also a good opportunity to replace the non-descriptive `x` list with destructuring and descriptive names. Like

    ([key, value, is_user]) => serviceRow(key, ..., is_user)

This makes it easier for a developer to follow the code what happens here.